packages/ui/src/components/Switch/base/SwitchBase.tsx

```diff
@@ -16,27 +16,28 @@ export interface SwitchBaseProps {
  * @constructor
  */
 export function SwitchBase(props: SwitchBaseProps) {
-  const {size = 'l', classes, activeColor, inactiveColor} = props;
+  const {size, activeColor, inactiveColor} = props;
   const [checked, setChecked] = useState(false);
   const [active, setActive] = useState(false);
 
   const onChange = useCallback(() => setChecked(state => !state), []);
   const disable = useCallback(() => setActive(false), []);
   const enable = useCallback(() => setActive(true), []);
 
-  const isLarge = size === 'l';
+  const classes = props.classes as (SwitchClasses & Record<string, string>);
+  const s = size?.toUpperCase();
```

Comment:
`size` should always have default value. In previous revision of this code, it was `l`. As long as new sizes were created, we should use `m` now as default one. Additionally, according to this code, `s` can become undefined which is incorrect in case, when it is used in future class names calculations (`classes['rootSize${s}']`, for example).